src/views/Options.vue

```diff
@@ -90,6 +90,7 @@
                 :label="$t('options.stay_logged_in')"
                 color="grey darken-1"
                 :input-value="stayLoggedIn"
```

Comment:
`input-value` prop has been renamed to `v-model` in Vuetify 3, so you can remove this line.

Vuetify 2
https://v2.vuetifyjs.com/en/api/v-checkbox/#props-input-value
Vuetify 3
https://vuetifyjs.com/en/api/v-checkbox/#props-model-value